src/components/Garden.js

```diff
@@ -0,0 +1,143 @@
+    .attr('d', petalPathTwo)
+    .attr('fill', d => colorsByLanguage[d.languages[1]] || 'none')
+    .attr('transform', `translate(-50, -150)rotate(60 50 130)`)
+  petalGroup.append('path')
+    .attr('d', petalPathTwo)
+    .attr('fill', d => colorsByLanguage[d.languages[1]] || 'none')
+    .attr('transform', `translate(-50, -150)rotate(180 50 130)`)
+  petalGroup.append('path')
+    .attr('d', petalPathTwo)
+    .attr('fill', d => colorsByLanguage[d.languages[1]] || 'none')
+    .attr('transform', `translate(-50, -150)rotate(300 50 130)`)
+  petalGroup.append('path')
+    .attr('d', petalPathOne)
+    .attr('fill', d => colorsByLanguage[d.languages[0]] || 'none')
+    .attr('transform', `translate(-50, -130)`)
+  petalGroup.append('path')
+    .attr('d', petalPathOne)
+    .attr('fill', d => colorsByLanguage[d.languages[0]] || 'none')
+    .attr('transform', `translate(-50, -130)rotate(120, 50, 110)`)
+  petalGroup.append('path')
+    .attr('d', petalPathOne)
+    .attr('fill', d => colorsByLanguage[d.languages[0]] || 'none')
+    .attr('transform', `translate(-50, -130)rotate(240,  50, 110)`)
+
+  //   DRAW STEMS
+  //   height scaled to activeLife
+    const stem = flowerBed.selectAll('.stem')
+      .data(repositories).enter()
+      .append('path')
+      .attr('class','stem')
+      .attr('id', (d, i) => `myStem${i}`)
+      .attr('d', d => `M0,600 C 80 ${yStemScale(d.lifespan)}, -20 ${yStemScale(d.lifespan)}, 0 ${yStemScale(d.lifespan)}`)
+      .attr('transform', (d, i) => `translate(${100 + i * 200},0)`)
+      .attr('stroke-width', 5)
+      .attr('stroke', 'green')
+      .attr('fill', 'none')
+
+      const animate = () => {
+        flowerBed.selectAll('.stem')
+          .transition().duration(2000)
+          .attr('d', d => `M0,600 C -20 ${yStemScale(d.lifespan)}, 20 ${yStemScale(d.lifespan)}, 0 ${yStemScale(d.lifespan)}`)
+            .attr('stroke', 'green')
+          .transition().duration(2000)
+          .attr('d', d => `M0,600 C 80 ${yStemScale(d.lifespan)}, -20 ${yStemScale(d.lifespan)}, 0 ${yStemScale(d.lifespan)}`)
+          .on("end", animate)
+      }
+    animate();
+
+
+    flowerBed.selectAll('.repoName')
+      .data(repositories).enter()
+      .append('text')
+      .append('textPath')
+      .attr('xlink:href', (d, i) => `#myStem${i}`)
+      .text(d => d.name)
+      .attr('font-size', '1.5rem')
+
+    const flowerCenter = flowerBed.selectAll('circle')
+      .data(repositories).enter().append('circle')
+      .attr('r', 25)
+      .attr('cx', (d, i) => 100 + i * 200)
+      .attr('cy', d => yStemScale(d.lifespan) - 20)
+      .attr('stroke-width', 1)
+      .attr('stroke', 'blue')
+      .attr('fill', 'blue')
+      //Append the month names to each slice
+
+
+  //   console.log('STEMS', stem)
+  //
+  //   //DRAW ROOTS
+  //   //one root for each branch
+  //   //text path w branch name
+  //   // const root =
+  //   // console.log('ROOTS', roots)
+  }
+  //
+  useEffect(() => {
+      drawGarden()
```

Comment:
we may want to have multiple functions nested in `drawGarden()` ? For readability? That said, a lot of the d3 examples I see throw SRP out the window